app/web/features/search/SearchResultsList.tsx

```diff
@@ -132,62 +139,72 @@ export default function SearchResultsList({
     resultsList = [{ user: selectedUserData.data, rank: 0, snippet: "" }];
   }
 
+  const resultsSnippet =
+    resultsList &&
+    resultsList.map((result) => (
+      <SearchResult
+        id={`search-result-${result.user!.userId}`}
+        className={classes.searchResult}
+        key={result.user!.userId}
+        user={result.user!}
+        onSelect={() => {
+          setSelectedResult({
+            userId: result.user!.userId,
+            lng: result.user!.lng,
+            lat: result.user!.lat,
+          });
+        }}
+        highlight={
+          selectedResult && selectedResult.userId === result.user!.userId
+        }
+      />
+    ));
+
   return (
-    <Paper className={classes.mapResults}>
+    <>
+      {/* SHOW VERTICAL LIST ON CLICK FOR MOBILE */}
+      {isMobile && resultsSnippet?.length && (
+        <SearchResultsMobileVerticalList resultsSnippet={resultsSnippet} />
+      )}
+
       {error && <Alert severity="error">{error}</Alert>}
+
       {!isMobile && (
-        <SearchBox
-          searchType={searchType}
-          setSearchType={setSearchType}
-          locationResult={locationResult}
-          setLocationResult={setLocationResult}
-          setQueryName={setQueryName}
-          queryName={queryName}
-        />
-      )}
-      <>
-        {isLoading ||
-          (selectedUserData.isLoading && (
-            <CircularProgress className={classes.baseMargin} />
-          ))}
+        <Paper className={classes.mapResults}>
+          <SearchBox
+            searchType={searchType}
+            setSearchType={setSearchType}
+            locationResult={locationResult}
+            setLocationResult={setLocationResult}
+            setQueryName={setQueryName}
+            queryName={queryName}
+          />
 
-        {!isLoading && !hasAtLeastOnePageResults && (
-          <TextBody className={classes.baseMargin}>
-            {t("search_result.no_user_result_message")}
-          </TextBody>
-        )}
+          {!isLoading && hasAtLeastOnePageResults && (
+            <Paper>
+              {isLoading ||
+                (selectedUserData.isLoading && (
+                  <CircularProgress className={classes.baseMargin} />
+                ))}
```

Comment:
Nit - turn the two isLoading conditions into a variable to make it more readable. Took me a minute to figure out what was happening here.